### PR TITLE
定性分析

### PR DESCRIPTION
**Pull request by conghaowoooong**
- [x] style特征分布
- [x] 风格提议-风格化预测分析
- [ ] ~~可视化分析~~ 跟随图一起修改

### CODE REVIEW

File: satoshi_aaai/contents/experiments.tex [hunk 1]
```diff
@@ -249,15 +249,22 @@ \subsubsection{Quantitative Analysis}
...
+The key design in \MODEL~is to hypothesize agents' potential behavior styles in the future.
+We visualize several sampled agents' $d$-dimension stylized feature $F$ (in \EQUA{eq_alpha_feature}) in some specific datasets to explore their distributions.
+We show the 2D feature distributions $P(F)$ on eth and zara1 in \FIG{fig_features} (a)(b), and the 3D distribution on univ in \FIG{fig_features}(c)(d)(e).
+These stylized features are hypothed via the $K_c = 10$ \MODEL.
+Each dot represents the feature of one proposed style.
+Features in different categories are distinguished with different colors.
+As shown in \FIG{fig_features}, features in each category have a specific spatial distribution.
+Specifically, all the features distribute on a low-dimensional manifold, and each category occupies a specific area in that manifold.
+It shows that different hidden categories' stylized features are sufficiently different in the semantic level.
+It means that \MODEL~could reflect agents' different styles of behavior patterns adaptively,  even though we do not know what these ``styles'' meanings for.
```
Comment (NorthOcean) [inline satoshi_aaai/contents/experiments.tex:262]: \MODEL~could reflect agents' different styles of behavior patterns adaptively,  even though we do not know the meanings of these ``styles''.
这会不会给自己埋坑，说这句话，只给出现象，没有给出解释。

File: satoshi_aaai/contents/experiments.tex [hunk 2]
```diff
@@ -289,54 +296,30 @@ \subsubsection{Qualitative analysis}
...
+
+\paragraph{Stylized Prediction}
+\MODEL's another significant design is to predict trajectories under the specific style.
+Specifically, we take agents' destination planning preferences to act as the reflection of their behavior styles.
+We show several predictions to the agent a and b with different styles given by \MODELD~in \FIG{fig_betamodelvis}.
+Predictions (yellow dots) show that \MODEL~has strong adaptability to different destination plannings (green dots).
+\MODEL~gives different shapes of predictions under various stylized destination plannings.
+It shows the effectiveness of the stylized prediction.
+On the other hand, these different predictions also show a better adaption to the scene constraints and social interactive behaviors.
+For example, it gives agents predictions like entering the shop in (b1), turn around in (a3)(b2), standing still in (b3), and going around the parking car in (a2).
```
Comment (NorthOcean) [inline satoshi_aaai/contents/experiments.tex:312]: turn - turning

File: satoshi_aaai/contents/experiments.tex [hunk 3]
```diff
@@ -289,54 +296,30 @@ \subsubsection{Qualitative analysis}
...
-Blue dots represent observed coordinates, and green dots are ground truth future coordinates.
-Yellow dots are deterministic predictions given by \MODELD~($K_c = 20$), and heatmaps are generative distributions given by \MODELG~($(K, K_c) = (10, 20)$).
-Results show that the two-stage multi-style \MODEL~has given multiple styles of predictions, such as pedestrians going ahead, turning around the crossroad, resting, and bicycles turning right or going ahead (but not turning left) at the roundabout.
+We display several prediction results of both \MODELD~and \MODELG~in \ETHUCY~and SDD  in \FIG{fig_vis}.
+Blue dots represent observed trajectories, and green dots are their ground truth future trajectories.
+We use yellow dots to show the deterministic predictions given by \MODELD~($K_c = 20$), and use heatmaps to show distributions of generative predictions given by \MODELG~($(K, K_c) = (10, 20)$).
+Results show that the multi-style \MODEL~has given multiple styles of predictions, such as pedestrians going ahead, turning around the crossroad, resting, and bicycles turning right or going ahead (but not turning left) at the roundabout.
```
Comment (NorthOcean) [inline satoshi_aaai/contents/experiments.tex:322]: bicycles - bicyclers

File: satoshi_aaai/contents/fig_features.tex [hunk 4]
```diff
@@ -14,14 +14,14 @@
     
 \begin{figure}[tbp]
 \centering
-\includegraphics[scale=0.25]{../figs/features.pdf}
+\includegraphics[scale=0.23]{../figs/features.pdf}
 \caption{
-    Feature visualization in \ALPHAMODEL~($K_c=10$).
-    Each dot represents one predicted feature that belongs to some hidden behavior category.
+    Stylized Feature Visualization.
+    Each dot represents one stylized feature that belongs to some hidden behavior category.
```
Comment (NorthOcean) [inline satoshi_aaai/contents/fig_features.tex:20]: some hidden behavior category - catrgories

File: satoshi_aaai/contents/fig_features.tex [hunk 5]
```diff
@@ -14,14 +14,14 @@
...
     Colors represent their categories.
-    (a), (b) are the 2-D feature distributions of dataset eth and zara1 via PCA.
-    univ has a larger number of trajectory data than others.
-    To better display its distribution, we show it in 3-D with (c), (d), and (e) three different views.
+    (a), (b) shows the 2D-distribution on eth and zara1 via PCA.
```
Comment (NorthOcean) [inline satoshi_aaai/contents/fig_features.tex:22]: shows - show

File: satoshi_aaai/contents/appendix_loss.tex [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+\begin{document}
+    
+\subfile{./fig_loss.tex}
+
+\section{Differential Displacement Loss}
+
+The differential displacement loss is used to make predictions in line with agents' physical constraints (especially for the generative \MODELG).
+As shown in \FIG{fig_loss}(a), prediction $P_1$ (DEF) and $P_2$ (GHI) have the same ADE to the ground truth (ABC).
+However, $P_1$ seems not so ``continuous'' like $P_2$.
```
[the remark this answers is not in the document]
Comment (conghaowoooong) [reply satoshi_aaai/contents/appendix_loss.tex:21]: `In reality`有点奇怪

File: satoshi_aaai/contents/experiments.tex (hunk 1 above)
Comment (conghaowoooong) [reply satoshi_aaai/contents/experiments.tex:262]: 重点在前边提到的`adaptively`上，怎么自适应的分类。
需要再考虑一下说法

File: satoshi_aaai/contents/experiments.tex (hunk 3 above)
Comment (conghaowoooong) [reply satoshi_aaai/contents/experiments.tex:322]: 重新看了SDD的原始annotations，已换成`Biker`

File: satoshi_aaai/contents/fig_features.tex (hunk 4 above)
Comment (conghaowoooong) [reply satoshi_aaai/contents/fig_features.tex:20]: 改成了
> Each dot represents one stylized feature that belongs to a specific hidden behavior category.

File: satoshi_aaai/contents/tab_ade_all.tex [hunk 7]
```diff
@@ -82,6 +80,7 @@
         Metrics are shown in the format of ``ADE/FDE'' in pixels.
         A lower value means a higher accuracy.
```
Comment (NorthOcean) [inline satoshi_aaai/contents/tab_ade_all.tex:81]: The low is better.

File: satoshi_aaai/contents/tab_ade_all.tex [hunk 8]
```diff
@@ -53,11 +50,12 @@
         Metrics are shown in the format of ``ADE/FDE'' in meters.
         A lower value means a higher accuracy.
```
Comment (NorthOcean) [inline satoshi_aaai/contents/tab_ade_all.tex:51]: The lower is better.

**Pull request merged**